scripts/Carousel.babel.js

```diff
@@ -0,0 +1,119 @@
+/*
+		@Autor  : Douglas Julião
+		@email  : douglas_juliao16@hotmail.com
+		@github : https://github.com/dougjuliao
+*/
+class Carousel {
+    constructor(config = {}) {
+        this.effect = config.effect || 'slideLeft';
+        this.width = config.width || '100%';
+        this.height = config.height || '250px';
+        this.selector = config.selector;
+        this.nav = config.nav || false;
+        this.time = config.time || 4;
+        this.slides = config.slides || [];
+        this.init();
+    }
+    addClass(selector, classe) {
+        selector.className += ' '+classe;
+    }
+    removeClass(selector, classe) {
+        selector.forEach(item => {
+            item.className = item.className.replace(classe, '');
+        });
+    }
+    setDimensions(selector) {
+        selector.style.width = this.width;
+        selector.style.height = this.height;
+    }
+    execEffect(selectorSlides) {
```

Comment:
I have the impression `execEffect` is doing more than its purpose by handling things like `setSlide`, `loop` and `loopStop` functions. What do you think? If yes, how could we improve it?

Reply:
Really yes, this function are a litle confuse i think it can be better with some adjusts, i will think in a better way.